src/app-prisma/prisma/schema.prisma

```diff
@@ -0,0 +1,18 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+model User {
+  id    Int     @id @default(autoincrement())
```

Comment:
Id to be UUID4

Reply:
this related to task number [36](https://2024-roadmap.atlassian.net/browse/VOID-36?atlOrigin=eyJpIjoiNmU3MjM5ODY3YjAxNGEwMGI3YTczMTBjNWVlY2JmMDkiLCJwIjoiaiJ9) in which will be handled